internal/alpini/netty4/alpini-netty4-base/src/test/java/com/linkedin/alpini/netty4/pool/TestChannelPoolManager.java

```diff
@@ -285,7 +285,7 @@ protected CallTracker createQueueBusyCallTracker(@Nonnull String queueName) {
 
       @Override
       void debugAcquireResolved(Future<Channel> promise) {
-        Time.sleepUninterruptably(1);
+        Time.sleepUninterruptably(100);
```

Comment:
This seems a bit of a jump, why the increase?

Reply:
Because the test is written quite poorly. It assumes that in the 1ms that this thread sleeps, the main test thread will go in and replace a global object. On github, there are only 2 VCPUs, and sometimes, it cannot achieve this.